composer.json

```diff
@@ -84,6 +84,7 @@
         "drupal/search_api": "^1.17",
         "drupal/viewfield": "^3.0@beta",
         "drupal/views_bulk_edit": "^2.4",
+        "drupal/views_field_view": "^1.0@beta",
```

Comment:
Are you comfortable using this module in beta? I know it's pretty widely used and I've used it on a number of D7 sites before, but not in 8.

Reply:
yeah. its been pretty actively maintained